sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -916,13 +913,18 @@ public virtual async Task<Response<ConfigurationSettingsSnapshot>> CreateSnapsho
             scope.Start();
             try
             {
-                RequestContext context = CreateRequestContext(ErrorOptions.Default, cancellationToken);
-                using RequestContent content = ConfigurationSettingsSnapshot.ToRequestContent(snapshot);
-                ContentType contentType = new ContentType(HttpHeader.Common.JsonContentType.Value.ToString());
+                //RequestContext context = CreateRequestContext(ErrorOptions.Default, cancellationToken);
+                //using RequestContent content = ConfigurationSettingsSnapshot.ToRequestContent(snapshot);
+                //ContentType contentType = new ContentType(HttpHeader.Common.JsonContentType.Value.ToString());
 
-                Response response = await CreateSnapshotAsync(name, content, contentType, context).ConfigureAwait(false);
-                ConfigurationSettingsSnapshot value = ConfigurationSettingsSnapshot.FromResponse(response);
-                return Response.FromValue(value, response);
+                //Response response = await CreateSnapshotAsync(name, content, contentType, context).ConfigureAwait(false);
+                //ConfigurationSettingsSnapshot value = ConfigurationSettingsSnapshot.FromResponse(response);
+                //return Response.FromValue(value, response);
+
+                // TODO: update to LRO pattern
```

Comment:
Do you have a bug for this? Or is this something you are planning to do as part of this PR itself?

Want to make sure this does not get missed.

Reply:
I'm working on it now, but I don't have access to corpnet for the time being, so nothing can be tested until the service is released.